src/ViewInstance.js

```diff
@@ -603,7 +603,7 @@ class ViewInstance extends React.Component {
                   onToggle();
                   this.handleViewSource(e, instance);
                 }}
-                disabled={!marcRecord}
+                isDisabled={!marcRecord}
```

Comment:
Please see the bug description's NOTE, it's better to rename the component's prop to just `disabled`, some action items also pass `disabled` instead of `isDisabled`